src/code/z_actor.c

```diff
@@ -2406,8 +2406,8 @@ void Actor_UpdateAll(PlayState* play, ActorContext* actorCtx) {
             } else if (!Object_IsLoaded(&play->objectCtx, actor->objectSlot)) {
                 Actor_Kill(actor);
                 actor = actor->next;
-            } else if ((requiredActorFlag && !(actor->flags & requiredActorFlag)) ||
-                       (!requiredActorFlag && canFreezeCategory &&
+            } else if ((freezeExceptionFlag && !(actor->flags & freezeExceptionFlag)) ||
+                       (!freezeExceptionFlag && canFreezeCategory &&
```

Comment:
use `freezeExceptionFlag == 0`/`freezeExceptionFlag != 0` since it's not a boolean?